src/TeacherRoster/TeacherRoster.js

```diff
@@ -13,7 +13,7 @@ const TeacherRoster = ({ students }) => {
   const renderStudentNames = () => {
     return students.map(student => {
       return (
-        <p id={student.id} onClick={(e) => findStudent(e)}>{student.first_name} {student.last_name}</p>
+        <p id={student.id} key={student.id} onClick={(e) => findStudent(e)}>{student.first_name} {student.last_name}</p>
       )
```

Comment:
This was really smart thinking - on click of the student, grab the id display the name.